frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -138,45 +138,143 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
   const [sortProperty, setSortProperty] = React.useState("name");
   const [sortOrder, setSortOrder] = React.useState("ascending");
 
+  const [grades, setGrades] = React.useState<Array<string>>([]);
+  const [testTypes, setTestTypes] = React.useState<Array<string>>([]);
+  const [countries, setCountries] = React.useState<Array<string>>([]);
+  const [regions, setRegions] = React.useState<Array<string>>([]);
+  const [status, setStatus] = React.useState("");
+
+  const countryOptions = [
+    { value: "Canada", label: "Canada" },
+    { value: "USA", label: "USA" },
+  ];
+
+  const regionOptions = [
+    "Ottawa",
+    "Calfornia",
+    "Ontario",
+    "Texas",
+  ].map((value) => ({ value, label: value }));
+
+  const testTypeOptions = [
+    { value: UseCase.BEGINNING, label: "Beginning" },
+    { value: UseCase.END, label: "End" },
+  ];
+  const gradeOptions = [
```

Comment:
can you merge the changes you made in your other branch and refactor to use the general constant?

Reply:
As soon as GradeEnum is merged, I will make that change!

Reply:
Done!